pygmo/_island_test.py

```diff
@@ -309,7 +309,7 @@ def run_io_tests(self):
         isl = island(algo=de(), prob=rosenbrock(), size=25)
         self.assertTrue(repr(isl) != "")
         self.assertTrue(isl.get_name() == "Thread island")
-        self.assertTrue(isl.get_extra_info() == "")
+        self.assertTrue(isl.get_extra_info() == "\tUsing pool: yes")
```

Comment:
Can you make this test conditional on the C++ pagmo version? This is done elsewhere in the test suite, e.g.:

https://github.com/esa/pygmo2/blob/master/pygmo/test.py#L633

Here you would have to check if the pagmo version is >= 2.16.